### PR TITLE
Add full MediaMetadata support in both Opera and Chrome

### PR DESCRIPTION
**Pull request by TheDude53**
This was tested in Chrome 83 and Opera 68 on Windows 10 64-bit.

### CODE REVIEW

File: api/MediaMetadata.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": true
```
Comment (jpmedley) [inline api/MediaMetadata.json:8]:
```suggestion
            "version_added": "57"
```

Comment (jpmedley) [reply api/MediaMetadata.json:8]: Please add this number to WebView also. Please do so for every place I change the Chrome value.

File: api/MediaMetadata.json [hunk 2]
```diff
@@ -53,7 +53,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata/MediaMetadata",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
```
Comment (jpmedley) [inline api/MediaMetadata.json:56]:
```suggestion
              "version_added": "57"
```

File: api/MediaMetadata.json [hunk 3]
```diff
@@ -101,7 +101,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata/album",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
```
Comment (jpmedley) [inline api/MediaMetadata.json:104]:
```suggestion
              "version_added": "57"
```

File: api/MediaMetadata.json [hunk 4]
```diff
@@ -149,7 +149,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata/artist",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
```
Comment (jpmedley) [inline api/MediaMetadata.json:152]:
```suggestion
              "version_added": "57"
```

File: api/MediaMetadata.json [hunk 5]
```diff
@@ -197,7 +197,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata/artwork",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
```
Comment (jpmedley) [inline api/MediaMetadata.json:200]:
```suggestion
              "version_added": "57"
```

File: api/MediaMetadata.json [hunk 6]
```diff
@@ -245,7 +245,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata/title",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
```
Comment (jpmedley) [inline api/MediaMetadata.json:248]:
```suggestion
              "version_added": "57"
```

**Comment by ddbeck** [comment]
Hi @TheDude53 do you plan to revisit this PR?

**Comment by TheDude53** [comment]
To be honest I forgot about it, but at least I remembered.

**Pull request merged**